pkgs/development/libraries/libctl/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, guile
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libctl";
+  version = "4.5.1";
+
+  src = fetchFromGitHub {
+    owner = "NanoComp";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "uOydBWYPXSBUi+4MM6FNx6B5l2to7Ny9Uc1MMTV9bGA=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook guile pkg-config ];
+
+  meta = with lib; {
+    description = "A Guile-based library for supporting flexible control files in scienti
+fic simulations.";
```

```suggestion
    description = "Guile-based library for supporting flexible control files in scientific simulations";
```

See [Submitting Changes](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes).

---

Thank you for reviewing my PR.